### PR TITLE
added clockworksms as provider

### PR DESCRIPTION
**Pull request by sourcefactory-at**
Added https://www.clockworksms.com/ as provider.

### CODE REVIEW

File: lib/Command/Configure.php [hunk 1]
```diff
@@ -135,6 +135,17 @@ private function configureSms(InputInterface $input, OutputInterface $output) {
 				$providerConfig->setUser($username);
 				$providerConfig->setPassword($password);
 
+				break;
+			case 'clockworksms':
+				$config->setProvider($provider);
+				/** @var WebSmsConfig $providerConfig */
```
Comment (ChristophWurst) [inline lib/Command/Configure.php]: This should be `ClockworkSMSConfig`

**Comment by sourcefactory-at** [comment]
Thanks, it works fine in my installation. Sorry about the wrong class name in the type hint comment. I checked it a few times before commiting but did not see that.

**Comment by ChristophWurst** [comment]
No worries, happens to me all the time :wink:

**Comment by ChristophWurst** [comment]
Released https://apps.nextcloud.com/apps/twofactor_gateway/releases?platform=14#14 :fireworks: :champagne:  Thanks again!

**Comment by sourcefactory-at** [comment]
Great, thanks for the update :)

**Pull request merged**